examples/images_contours_and_fields/interpolation_methods.py

```diff
@@ -1,18 +1,23 @@
-'''
-Show all different interpolation methods for imshow
-'''
+"""
+=================================
+Interpolations for imshow/matshow
+=================================
 
-import matplotlib.pyplot as plt
-import numpy as np
+This example displays the difference between interpolation methods for imshow
+and matshow.
+
+If `intperpolation` is None, it defaults to the rc image.interpolation
+parameter. If the interpolation is `none`, then no interpolation is performed
+for the Agg, ps and pdf backends. Other backends will defult to 'nearest'.
 
-# from the docs:
+For the Agg, ps and pdf backends, interpolation = 'none' works well when a big
+image is scaled down, while interpolation = 'nearest' works well when a small
+image is blown up.
```

Comment:
large image is scaled down, small image is scaled up. May as well keep the language parallel.